CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## next
 
+## 1.2.0
+* New showHeaderTabs prop that can be used to hide tabs in the header (e.g. in a single step wizard)
```

Comment:
Petty thing, but shouldn't those be under  '## next' ?

Reply:
Now to come think of it, you are right!  I fixed it.